public/locales/en-US/send.ftl

```diff
@@ -43,16 +44,21 @@ deleteFileButton = Delete file
 sendAnotherFileLink = Send another file
     .title = Send another file
 
+// This alt text and the download filename string are actions.
 downloadAltText
     .alt = Download
 downloadFileName  = Download { $filename }
+
 downloadFileSize = ({ $size })
 downloadMessage = Your friend is sending you a file with Firefox Send, a service that allows you to share files with a safe, private, and encrypted link that automatically expires to ensure your stuff does not remain online forever.
+
+// This is also an action.
```

Comment:
Alternative text used on the download link/button (indicates an action).

(localizers don't see the whole file, only one string at a time)